Example/RSKImageCropperExampleTests/RSKImageCropViewControllerTests.m

```diff
@@ -1018,6 +1034,26 @@ - (void)resetZoomScale;
     });
 });
 
+describe(@"zoomToRect", ^{
+    before(^{
+        imageCropViewController = [[RSKImageCropViewController alloc] initWithImage:originalImage];
+        sharedLoadView();
+    });
+    
+    it(@"zooms the image", ^{
```

Comment:
Actually, we should test how `imageCropViewController` zooms to a specific area of the image.
So, it will be something like that:
```obj-c
it(@"zooms to a specific area of the image", ^{
    CGRect rect = CGRectMake(100.0, 100.0, 400.0, 400.0);
    [imageCropViewController zoomToRect:rect animated:NO];
    
    UIScrollView *imageScrollView = imageCropViewController.imageScrollView;
    CGRect visibleRect = CGRectMake(round(imageScrollView.contentOffset.x / imageScrollView.zoomScale),
                                    round(imageScrollView.contentOffset.y / imageScrollView.zoomScale),
                                    imageScrollView.bounds.size.width / imageScrollView.zoomScale,
                                    imageScrollView.bounds.size.height / imageScrollView.zoomScale);
    
    BOOL contains = CGRectContainsRect(rect, visibleRect);
    expect(contains).to.beTruthy();
});
```